pasta_eln/handleDictionaries.py

```diff
@@ -38,6 +38,32 @@ def ontology2Labels(ontology:dict[str,Any], tableFormat:dict[str,Any]) -> dict[s
   return dataDict
 
 
+def ontologyV2_to_V3(ontology: dict[str, Any]) -> None:
```

Comment:
Is it not possible to import the function from ontology_configuration? why do we have to duplicate?

Reply:
There is a CLI version (cli.py) that works without Qt. Hence this translator should also work without Qt and the GUI. Therefore, I create a version that does not require Qt anything in the function header. Does that make sense?

I also thought of moving the translator into the higher level folder, but then all your imports are broken, and I did not want to do that. 
What would you suggest of how we proceed?

Reply:
Ahh Okay, if it's not simply easy to import and use it straightaway, I would suggest to keep it like this and proceed..